testutil/minio.go

```diff
@@ -20,12 +20,7 @@ import (
 	"github.com/minio/minio-go/v6"
 )
 
-var (
-	accessKey = "accesskey"
-	secretKey = "secretkey"
-)
-
 // NewMinioClient returns a minio client.
 func NewMinioClient() (*minio.Client, error) {
-	return minio.New(MinioInstance, accessKey, secretKey, false)
+	return minio.New(MinioInstance, "accesskey", "secretkey", false)
```

Comment:
Nit - I'd rather keep it embedded at the top. So that it can be reused and is more readable.

Reply:
nit#2 - can we keep one source of truth i.e. https://github.com/dgraph-io/dgraph/blob/aman/minio/systest/backup.env ...

Reply:
This is gonna require more efforts and time. I can work on this separately.

Reply:
lets not spend time on this .. I agree it is going to take a lot of time fixing this. It's better we do this when we cleanup all these docker-compose file crufts or something once we stabilize.